src/Linux/README.MD

```diff
@@ -73,6 +73,19 @@ build-essential package (`sudo apt install build-essential`) and dh-virtualenv
 
 The output deb file is suitable for installation on any amd64 Ubuntu system.
 
+In Ubuntu 20.04, please try this command to install dh-virtualenv:
```

Comment:
Why are these required? (dh-virtualenv and fakeroot)?
dh-virtualenv looks like it is for python
Can fakeroot be avoided if the user has root privlege?

I'm not a familiar with either of these tools, so please let me know if there are new requirements specific to Ubuntu 20.04 which are requiring them.